README.md

```diff
@@ -61,23 +51,13 @@ ABCVAEWFOAFW...
 -----END OPENSSH PRIVATE KEY-----
 ```
 
-##### `infrastructure.json:`
+##### `infrastructure.js:`
 
 ```
 module.exports = {
   servers: [
-    {
-      description: 'server',
-      name: 'server1',
-      domain: 'mydomain.mymachines.com',
-      port: '3000'
-    },
-    {
-      description: 'server',
-      name: 'server2',
-      domain: 'mydomain.mymachines.com',
-      port: '3000'
-    },
+    { host: 'server1', address: 'mydomain.mymachines.com' },
```

Comment:
Shouldn't it be `{ name: ..., address: ... }` for the UI to display?

Reply:
Nice catch :+1: